src/SamplesApp/UITests.Shared/Windows_UI_Xaml/MessageDialog/MessageDialog_Simple.xaml.cs

```diff
@@ -0,0 +1,158 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using System.Threading;
+using System.Threading.Tasks;
+using Uno.UI.Samples.Controls;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI.Popups;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+
+// The User Control item template is documented at https://go.microsoft.com/fwlink/?LinkId=234236
+
+namespace UITests.Shared.Windows_UI_Xaml.MessageDialog
+{
+	[SampleControlInfo("MessageDialog", "MessageDialog_Simple")]
+	public sealed partial class MessageDialog_Simple : UserControl
+	{
+		public MessageDialog_Simple()
+		{
+			this.InitializeComponent();
+		}
+
+		private async void ShowDialog(object sender, RoutedEventArgs e)
+		{
+			var response = await Show(CancellationToken.None, "Are you sure?", "Confirmation");
+
+			if (response == "Yes")
+			{
+				Show("Action executed.");
+			}
+			else
+			{
+				Show("Action canceled.");
+			}
+		}
+
+		private async void ShowComplexDialog(object sender, RoutedEventArgs e)
+		{
+			var choices = new[]
+			{
+				"Captain Kirk",
+				"Han Solo",
+				"Neo"
+			};
+
+			var dialog = new Windows.UI.Popups.MessageDialog("Which one is your preferred character?", "Quiz");
+
+			try
+			{
+				for (int i = 0; i < choices.Length; i++)
+				{
+					dialog.Commands.Add(new UICommand(choices[i], null, i));
+				}
+			}
+			catch (Exception error)
+			{
+				// On Windows Phone, this will fail. If you add more than three, also on Windows and Android.
+				Show(error.Message);
+				return;
+			}
+
+			dialog.DefaultCommandIndex = 0;
+
+			var response = await dialog.ShowAsync();
+
+			if (response?.Id == null)
+			{
+				Show("You don't want to commit yourself? Fine...");
+			}
+			else
+			{
+				Show($"You chose {choices[(int)response.Id]}, which is a very good pick!");
+			}
+		}
+
+		private async void ShowDialogs(object sender, RoutedEventArgs e)
+		{
+			var response1 = await Show(CancellationToken.None, "This is the first question.", "Important");
+			var response2 = await Show(CancellationToken.None, "This is the second question.", "Warning", "Ok", "Cancel");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4826544)